.github/workflows/pr-checks.yml

```diff
@@ -14,6 +14,8 @@ jobs:
           - 2.12.19
           - 2.13.13
           - 2.13.14
+          - 3.3.3
+          - 3.4.2
```

Comment:
My github action-voo is not amazing, probably want to only run Scala 3 with 11. If people are modern enough to use Scala 3, then surely they moved on from Java 8.